osu-capstone-cs72/cs-applied-plan-portal
# Redirect to manage role

This PR includes a page for head advisers to manage the roles of the users in the database. There is an additional button in the navbar to go to the "Manage roles" page, from which the head adviser will search for people and use the drop down menu to change the role. There will be a confirmation message before this change is made. 

Steps to test:

- Search for a user in the database using "Any role" or the role they are assigned and make sure the correct results return
- Search for a user with a filter that does not include their role and ensure they do not appear
- Search for a random string and ensure a warning appears saying "no users found"
- Change a users role, click yes on the confirmation message and check the database to make sure the change was made
- Try to change a users role and click no on the confirmation message and make sure nothing happens
- Change your own role in the database to either 0 or 1 to make sure the "Manage roles" button does not appear in the navbar
